models/demos/deepseek_v3/tt/rms_norm/distributed_rms_norm.py

```diff
@@ -104,7 +107,27 @@ def _model_config(
         cls, hf_config: PretrainedConfig, mesh_device: ttnn.Device, rms_norm_stats_memory_config: ttnn.MemoryConfig
     ) -> dict[str, OpConfigBase]:
         """Generate model configuration for RMSNorm."""
+
+        shard_core_grid = ttnn.CoreGrid(x=4, y=7)
```

Comment:
do we not define this memory config in rms_norm_base already? Why is this that code duplicated here?